### PR TITLE
finished producer

### PR DESCRIPTION
**Pull request by joao-r-reis**

### CODE REVIEW

File: src/Logging.Producer/LogEventExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                MessageTemplate = @this.MessageTemplate.Text,
+                Timestamp = @this.Timestamp,
+                RenderedMessage = @this.MessageTemplate.Render(@this.Properties),
+                Properties = @this.Properties.ToDictionary(kv => kv.Key, kv => kv.Value.ToString())
+            };
+        }
+
+        private static LogLevel MapToLogLevel(this LogEventLevel @this)
+        {
+            switch (@this)
```
Comment (pedromsmoreira) [inline src/Logging.Producer/LogEventExtensions.cs]: What if we mapped this LogEventLevels into a Dictionary<LogEventLevel, LogLevel>?

File: test/unit/Logging.Consumer.Tests/StyleCop.Cache [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<stylecopresultscache>
```
Comment (pedromsmoreira) [inline test/unit/Logging.Consumer.Tests/StyleCop.Cache]: Add StyleCop.cache to .gitignore, we just need the `StyleCop.Settings` file in the solution.

File: src/Logging.Producer/LogEventExtensions.cs (hunk 1 above)
Comment (joao-r-reis) [reply src/Logging.Producer/LogEventExtensions.cs]: Done

File: test/unit/Logging.Consumer.Tests/StyleCop.Cache (hunk 2 above)
Comment (joao-r-reis) [reply test/unit/Logging.Consumer.Tests/StyleCop.Cache]: Had to remove one line from gitignore which was making git not ignore stylecop.cache

**Comment by pedromsmoreira** [comment]
👍

**Pull request merged**